.env.example

```diff
@@ -24,6 +24,9 @@ ELASTICSEARCH_MESSAGE_INDEX = messages
 REDIS_URL = redis://:redis@ao-cache-redis:6379
 NEXTAUTH_SECRET = 123
 
+# Use this if deploying in a staging enviroment
+# STAGING = "1"
```

Comment:
change from stagining 1 to DEPOYMENT_ENV, options are development, staging, production